pkg/common/resource.go

```diff
@@ -79,6 +79,21 @@ func GetPodResource(pod *v1.Pod) (resource *si.Resource) {
 		checkInitContainerRequest(pod, podResource)
 	}
 
+	// K8s pod EnableOverHead from:
+	// alpha: v1.16
+	// beta: v1.18
+	// Enables PodOverhead, for accounting pod overheads which are specific to a given RuntimeClass
+
+	// If Overhead is being utilized, add to the total requests for the pod
```

Comment:
you are skipping BestEffortQOS pods as they have already returned earlier in the function. A BestEffortQOS pod can have init containers filed a follow up jira for QOS BestEffort pod improvements

Reply:
Sure @wilfred-s , i see the follow-up now, i will add the corresponding things in
https://issues.apache.org/jira/browse/YUNIKORN-1559